<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QWidget" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>650</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>850</width>
    <height>650</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>850</width>
    <height>650</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#m_close{border-image:url(:/res/close .png);}
QWidget#m_setting{border-image:url(:/res/setting.png);}
QWidget#m_backgroud1{border-image:url(:/res/backgroud.webp);}
QWidget#m_backgroud2{color:rgb(255,255,255);}
QWidget#m_tip{color:red;}
</string>
  </property>
  <widget class="QWidget" name="m_backgroud2" native="true">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>0</y>
     <width>350</width>
     <height>650</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>350</width>
     <height>650</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>350</width>
     <height>650</height>
    </size>
   </property>
   <widget class="QWidget" name="m_backgroud3" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>350</width>
      <height>280</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>350</width>
      <height>280</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>350</width>
      <height>280</height>
     </size>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <widget class="QPushButton" name="m_close">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>0</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="m_setting">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>0</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_3" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>280</y>
      <width>280</width>
      <height>120</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>280</width>
      <height>120</height>
     </size>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLineEdit" name="m_user">
       <property name="minimumSize">
        <size>
         <width>280</width>
         <height>60</height>
        </size>
       </property>
       <property name="placeholderText">
        <string>用户名/账号/邮箱</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="m_passwd">
       <property name="minimumSize">
        <size>
         <width>280</width>
         <height>60</height>
        </size>
       </property>
       <property name="placeholderText">
        <string>密码</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="widget_4" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>430</y>
      <width>350</width>
      <height>60</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="m_login">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>登录</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="m_register">
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>注册</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="m_tip">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>500</y>
      <width>280</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="m_backgroud1" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>650</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
